crates/vsock/src/main.rs

```diff
@@ -111,6 +125,7 @@ fn parse_vm_params(s: &str) -> Result<VsockConfig, VmArgsParseError> {
     let mut socket = None;
     let mut uds_path = None;
     let mut tx_buffer_size = None;
+    let mut group = "default";
```

Comment:
I don't have a strong opinion, but since for example for `guest_cid` and `tx_buffer_size` we used Option and then `unwrap_or`, what about doing the same for `group`?

Reply:
Also, can we use `DEFAULT_GROUP_NAME`?